OpenBVE.sln

```diff
@@ -34,6 +34,7 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OpenBve", "source\OpenBVE\O
 		{678031C7-6D87-4F5F-9C4A-12F86B76342C} = {678031C7-6D87-4F5F-9C4A-12F86B76342C}
 		{B520B1D7-3889-4C88-9E0F-CB96802D5CD1} = {B520B1D7-3889-4C88-9E0F-CB96802D5CD1}
 		{F714A9FA-1C4B-456B-BACF-D6763223A81C} = {F714A9FA-1C4B-456B-BACF-D6763223A81C}
+		{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC} = {FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}
```

Comment:
Need to make sure the thing rebuilds as a dependancy of Route Viewer / Main program